docs/api/saved-objects/import.asciidoc

```diff
@@ -22,8 +22,17 @@ experimental[] Create sets of {kib} saved objects from a file created by the exp
 [[saved-objects-api-import-query-params]]
 ==== Query parameters
 
+`createNewCopies`::
+  (Optional, boolean) Creates new copies of saved objects, regenerating each object's ID and resetting its origin in the process. If this
+  option is used, potential conflict errors will be avoided.
++
+NOTE: This cannot be used with the `overwrite` option.
+
 `overwrite`::
-  (Optional, boolean) Overwrites saved objects.
+  (Optional, boolean) Overwrites saved objects if they already exist. If this option is used, potential conflict errors will be
+  automatically resolved by overwriting the destination object.
++
+NOTE: This cannot be used with the `createNewCopies` option.
```

Comment:
Do we need to specify that this is deprecated? If so, what's the best way to do it? I searched through the other Asciidoc files and couldn't find any convention for this.

Reply:
I don't think it's strictly necessary, since all of our APIs are either marked as beta or experimental. We don't offer guarantees for API stability at this point